x-pack/plugins/security_solution/public/common/hooks/search_bar/use_init_timerange_url_params.ts

```diff
@@ -68,7 +83,7 @@ const initializeTimerangeFromUrlParam = (
         dispatch(
           inputsActions.setAbsoluteRangeDatePicker({
             ...absoluteRange,
-            id: timelineId,
+            id: 'timeline',
```

Comment:
why not use the constants?

Reply:
they're not required. i will be updating these in a follow up pr with a [dedicated type](https://github.com/elastic/kibana/pull/139508/commits/42b56b55982dfd9c87a0d63207e7f48432016bac) so it doesn't matter anyways, will change soon